env/env_encryption.cc

```diff
@@ -362,6 +362,15 @@ void EncryptedWritableFile::PrepareWrite(size_t offset, size_t len,
   file_->PrepareWrite(offset + prefixLength_, len, options, dbg);
 }
 
+void EncryptedWritableFile::SetPreallocationBlockSize(size_t size) {
+  file_->SetPreallocationBlockSize(size);
+}
```

Comment:
Is this correct?  I thought all of the encrypted files were written in more certain block sizes with potentially a long header.  Is this keeping with that?  It might be correct, but a comment/explanation would be helpful...

Reply:
Added comment for that.
Size here is the preallocation size, there's no need to add or remove `prefixLength_`. `PrepareWrite()` will use `preallocation size` to preallocate if needed, which is including the `prefixLength_`.